docker/Dockerfile

```diff
@@ -22,8 +22,8 @@ ARG GATEWAY_GITHUB=https://github.com/oasisprotocol/oasis-rosetta-gateway
 ARG JEMALLOC_VERSION=5.2.1
 ARG JEMALLOC_CHECKSUM=34330e5ce276099e2e8950d9335db5a875689a4c6a56751ef3b1d8c537f887f6
 
-ARG GENESIS_MAINNET_URL=https://github.com/oasisprotocol/mainnet-artifacts/releases/download/2021-04-28/genesis.json
-ARG GENESIS_MAINNET_FILE_CHECKSUM=8ae33cffe67e3df1be392dddde332a73f207cede5992c93600b0ae65c22af2a2
+ARG GENESIS_MAINNET_URL=https://github.com/oasisprotocol/mainnet-artifacts/releases/download/2022-04-11/genesis.json
```

Comment:
Let's note the Mainnet genesis file update for the Damask upgrade in a separate commit and give it a Change Log fragment.

Reply:
unsquashed, added changelog

Reply:
Thanks!